.github/workflows/go-pr.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: '1.22.7'
```

Comment:
Can we not pin to .7? can we use 1.22 instead?

Reply:
Will try with 1.21 to match the go.mod

Reply:
It don't need to match go.mod, just need to be equal or higher.